packages/synapse-react-client/src/synapse-queries/entity/useExportToCavatica.ts

```diff
@@ -24,9 +24,10 @@ export function useExportToCavatica(
     try {
       // add drs_uri to select
       const selectColumnsList = selectColumns
-        ?.map(col => `"${col.name}"`)
+        ?.filter(col => col.name != 'name')
```

Comment:
Filter out any existing select column called 'name', since we will be adding the 'name' column in our sql to generate the manifest.

Reply:
may want to add a test for the filter